[2021-02-08 15:47:13 - INFO - transport.py:line 1819]: Connected (version 2.0, client OpenSSH_7.4)
[2021-02-08 15:47:14 - INFO - transport.py:line 1819]: Authentication (publickey) failed.
[2021-02-08 15:47:14 - INFO - transport.py:line 1819]: Connected (version 2.0, client OpenSSH_7.4)
[2021-02-08 15:47:14 - INFO - transport.py:line 1819]: Authentication (password) successful!
